### ISSUE TITLE
Datatest not show in Serentiy Report when using @CsvFileSource

### ISSUE THREAD

**Issue by ndagnhat** [serenity-bdd/serenity-core#3007]
My code
![image](https://user-images.githubusercontent.com/65029229/211260598-633b6f39-0f9e-43dd-ac06-3ba64b9a1e60.png)

CSV file
![image](https://user-images.githubusercontent.com/65029229/211260720-cd2d2b18-4964-4fdf-8436-e16e2012b682.png)

Serenity Report
![image](https://user-images.githubusercontent.com/65029229/211260758-36238c4a-e708-4d7a-bdfe-955ebd8ed9c4.png)

**Comment by zzoubian** [comment]
Hello @ndagnhat - I was able to reproduce something similar to you when I changed it to "files", have you tried changing it to "resources" instead, this populated the example table for me.

@CsvFileSource(resources = "/testlist.csv", numLinesToSkip = 1)

**Comment by zzoubian** [comment]
Are you executing with maven or gradle?

I noticed something odd when running with gradle:
![image](https://user-images.githubusercontent.com/35345489/211640632-212b18e7-ff25-413b-9262-74a9fa8bcd00.png)

**Comment by ndagnhat** [comment]
> Hello @ndagnhat - I was able to reproduce something similar to you when I changed it to "files", have you tried changing it to "resources" instead, this populated the example table for me.
> 
> @CsvFileSource(resources = "/testlist.csv", numLinesToSkip = 1)
hi @zzoubian 
i have tried both ("files" or "resources") but data test doesn't show in Report
Thanks for reply

**Comment by zzoubian** [comment]
I'll give it a try on Windows, and see if I get the same error.

**Comment by zzoubian** [comment]
> > Hello @ndagnhat - I was able to reproduce something similar to you when I changed it to "files", have you tried changing it to "resources" instead, this populated the example table for me.
> > @CsvFileSource(resources = "/testlist.csv", numLinesToSkip = 1)
> 
> hi @zzoubian i have tried both ("files" or "resources") but data test doesn't show in Report, and when tried changing files to resources instead, this error(#2956) appear in console
> 
> Thanks for reply

You are correct, I see the same exception from #2956 and there is no data in the examples table when using resources on Windows. Even though, it runs through all the tests fine.

**Comment by wakaleo** [comment]
Can you dig into the code and see if you can propose a PR with a fix? The JUnit 5 integration is still a bit experimental in parts.

**Comment by zzoubian** [comment]
@wakaleo  - appears to be a simple fix, adding .getClassLoader() to lines 99 (getDataFileFor) and 109 (isAClassPathResource) in JUnit5CSVTestDataSource.java, fixes the issue for me.

**Comment by wakaleo** [comment]
Can you propose a PR?

**Comment by zzoubian** [comment]
I'll need to do a little bit more digging, after the change, it broke on macos :) if i get it working for both OS, i'll create a PR.
